source/core/src/main/com/csse3200/game/rendering/AnimationRenderComponent.java

```diff
@@ -179,7 +179,7 @@ protected void draw(SpriteBatch batch) {
 
   @Override
   public void dispose() {
-    atlas.dispose();
+   // atlas.dispose(); // this has to be disabled to keep the atlas file for other entities that rely on it
```

Comment:
Should probably just remove this line - since you've fixed this bug, there should be no reason to keep it